### PR TITLE
add feature/arrays challenge

### PR DESCRIPTION
**Pull request by Aitor1111**
Add finished challenge of making array methods "a manija"

### CODE REVIEW

File: staff/aitor-trujillo/arrays/array-functions/every.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe("myEvery", function () {
```
Comment (manuelbarzi) [inline staff/aitor-trujillo/arrays/array-functions/every.spec.js:1]: 'every'

File: staff/aitor-trujillo/arrays/array-functions/filter.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function myFilter(array, callback) {
```
Comment (manuelbarzi) [inline staff/aitor-trujillo/arrays/array-functions/filter.js:1]: filter

File: staff/aitor-trujillo/arrays/array-functions/every.spec.js (hunk 1 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/arrays/array-functions/every.spec.js]: Solved the format error, from my<Function> to <function>. Thanks for the review! 👍

File: staff/aitor-trujillo/arrays/array-functions/filter.js (hunk 2 above)
Comment (Aitor1111) [reply staff/aitor-trujillo/arrays/array-functions/filter.js]: Solved the format error, from my<Function> to <function>. Thanks for the review! 👍

File: staff/aitor-trujillo/arrays/arroz/arroz.prototype.join.spec.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  });
+  it("should join all items of array into string and return it with separator comma (not the last one)", function () {
+    var myArroz = new Arroz("hello", 1, 2, "world");
+
+    var result = myArroz.join(".");
+
+    expect(typeof result).toBe("string");
+    expect(result.length).toBe(15);
+  });
+});
```
Comment (lichab) [inline staff/aitor-trujillo/arrays/arroz/arroz.prototype.join.spec.js:18]: You could test also that the default behaviour of separating with commas is working.

File: staff/aitor-trujillo/arrays/arroz/arroz.prototype.pop.spec.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+describe("pop", function () {
+  it("should delete the last number of array", function () {
+    var arroz = new Arroz(1, 2, 3);
+
+    arroz.pop();
+
+    expect(arroz[1]).toBe(2);
+    expect(arroz[2]).toBe(undefined);
+  });
+});
```
Comment (lichab) [inline staff/aitor-trujillo/arrays/arroz/arroz.prototype.pop.spec.js:10]: You could also test that length of the array was modified and that the returned value was actually the removed item.

File: staff/aitor-trujillo/arrays/arroz/arroz.prototype.pop.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+Arroz.prototype.pop = function () {
+  var remove = this[this.length - 1];
+
+  this[this.length--];
```
Comment (lichab) [inline staff/aitor-trujillo/arrays/arroz/arroz.prototype.pop.js:6]: You don't really need the enclosing `this`  here.  Simply `this.length--` would do the job.

**Pull request merged**